pytest-selenium/pages/content.py

```diff
@@ -108,4 +104,4 @@ def section_url(self):
 
         def click_attribution_link(self):
             self.offscreen_click(self.attribution_link)
-            return self.page.attribution.wait_for_region_to_display()
+            self.page.attribution.wait_for_region_to_display()
```

Comment:
Why was this return removed? It should return the page so that we have a reference to it in the code.

Reply:
@m1yag1 
I was discussing about some random locators with Greg and he suggested that a return is not necessary since we are already in that page

Reply:
ok then place a `return` only. you don't need to put anything after.

Reply:
@m1yag1 
do you mean just have the click function and remove the whole wait for region code
         def click_attribution_link(self):
             self.offscreen_click(self.attribution_link)

Reply:
Mike resolving this conservation to leave the code as such